user-interface/src/search/SearchScreen.test.tsx

```diff
@@ -56,67 +65,39 @@ describe('search screen', () => {
     });
     await waitFor(() => {
       // wait for loading to disappear
-      expect(document.querySelector('.loading-spinner')).not.toBeInTheDocument();
+      expect(loadingSpinner).not.toBeInTheDocument();
       table = document.querySelector('#search-results > table');
       expect(table).toBeVisible();
     });
     const rows = document.querySelectorAll('#search-results-table-body > tr');
     expect(rows).toHaveLength(caseList.length);
 
-    const caseNumber = '00-11111';
-    fireEvent.change(caseNumberInput, { target: { value: caseNumber } });
+    expect(getCaseSummarySpy).toHaveBeenLastCalledWith(
+      '/cases',
+      expect.objectContaining(divisionSearchPredicate),
+    );
+
+    // Make second search request...
+    fireEvent.click(expandButton!);
+    const chapterTwelveOptionButton = screen.getByTestId('combo-box-option-12');
+    fireEvent.click(chapterTwelveOptionButton);
+    fireEvent.click(expandButton!);
     await waitFor(() => {
       expect(document.querySelector('.loading-spinner')).toBeInTheDocument();
```

Comment:
Could we replace document.querySelector with 'loadingSpinner'

Reply:
Have to re-query because the screen is probably rerendered.